ansible/roles/commcarehq/tasks/main.yml

```diff
@@ -110,3 +110,26 @@
   tags:
     - localsettings
     - hq-localsettings
+
+- name: Copy celery restart script
+  sudo: yes
+  template:
+    src: "restart_celery.sh.j2"
+    dest: "/root/restart_celery.sh"
```

Comment:
Sounds good @javierwilson I was copying htis from the couch restarter scrpit. will be cleaning it up today